### PR TITLE
Apps: added Using `/apps debug` command

### PR DESCRIPTION
**Pull request by levb**
#### Summary
- added Using `/apps debug` command to debug Mattermost Apps.

for https://github.com/mattermost/mattermost-plugin-apps/pull/394

#### Ticket Link
n/a

### CODE REVIEW

**Comment by asatkinson** [comment]
is this going to ship with app framework 1.2 (next week)?

**Comment by asatkinson** [comment]
When this ships, will the support for testing with a cloud instance also work?

I think that if this means we can support cloud servers, we need to update the section on developer setup to show how to do this on this page "/integrate/apps/quickstart/"

This will be a huge improvement for community members, so we should highlight how to set this up.

File: site/content/integrate/apps/quickstart/apps-debug/_index.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+---
+title: "Using /apps debug command"
```
Comment (asatkinson) [inline site/content/integrate/apps/quickstart/apps-debug/_index.md:2]: What about "Debugging Apps" for the title

File: site/content/integrate/apps/quickstart/apps-debug/_index.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+heading: "Using /apps debug command"
+description: "`/apps debug` command can be used to debug an App's bindings, KV store records, view the logs, etc."
+weight: 100
+aliases:
+  - /integrate/apps/apps-debug/
+---
+
+This reference guide documents /apps debug command that can be used to display
```
Comment (asatkinson) [inline site/content/integrate/apps/quickstart/apps-debug/_index.md:10]: The /apps debug command is available to help developers debug problems with their apps.  This slash command can be used to provide debug logs right in a Mattermost channel, examine, change and delete data stored by your app and examine OAuth2 data.

This command is available exclusively when a Mattermost server is running developer mode.  Instructions for turning on developer mode are available <here> **TODO**

Comment (asatkinson) [reply site/content/integrate/apps/quickstart/apps-debug/_index.md:10]: The rest about the actual slash commands is great.  This is so cool!

**Comment by neflyte** [comment]
Is "Debugging Apps" part of the Apps Quickstart? I'm not convinced.
This topic should go in a seperate section; something about "troubleshooting". It doesn't make sense to be in the quickstart area.

**Review by neflyte — CHANGES_REQUESTED** [review]
Please move this document out of the quickstart section and into a new section for troubleshooting. We can discuss the location further with @asatkinson and @azigler if we can't decide on one.

File: site/content/integrate/apps/quickstart/apps-debug/_index.md (hunk 1 above)
Comment (cwarnermm) [reply site/content/integrate/apps/quickstart/apps-debug/_index.md:2]: Building on this great idea, let's go with "Debug apps" given that we avoid gerunds across the docs set.

File: site/content/integrate/apps/quickstart/apps-debug/_index.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+and modify apps' bindings, KV store records, and other data.
+
+To use this command you need to have both **Developer Mode** and **Testing Commands** set to `true`.
```
Comment (cwarnermm) [inline site/content/integrate/apps/quickstart/apps-debug/_index.md:13]: Let's be a bit more explicit about where, in-product, users can set these values.

**Review by cwarnermm — COMMENTED** [review]
Provided feedback inline.

File: site/content/integrate/apps/quickstart/apps-debug/_index.md (hunk 1 above)
Comment (neflyte) [reply site/content/integrate/apps/quickstart/apps-debug/_index.md:2]: In #1181 I've moved this page to the Functionality section.

File: site/content/integrate/apps/quickstart/apps-debug/_index.md (hunk 3 above)
Comment (neflyte) [reply site/content/integrate/apps/quickstart/apps-debug/_index.md:13]: In #1181 I've made this more clear.

**Comment by neflyte** [comment]
Closing PR in favour of #1181

**Pull request closed without merging**